src/meshapi/views/forms.py

```diff
@@ -101,6 +109,21 @@ def join_form(request: Request) -> Response:
         logging.exception("TypeError while processing JoinForm")
         return Response({"detail": "Got incomplete form request"}, status=status.HTTP_400_BAD_REQUEST)
 
+    if not settings.DEBUG and not DISABLE_RECAPTCHA_VALIDATION:
```

Comment:
Does this belong here? I put it here so that we don't need to worry about this validation in the join form replay CLI, but does that open us up to any risks?